### PR TITLE
replaces worker and api with old iron_go

### PR DESCRIPTION
**Pull request by bg451**
ports gerred's httpClient stuff over.

This might break stuff for some people, hopefully not.

### CODE REVIEW

**Comment by bg451** [comment]
ping @rdallman

**Comment by bg451** [comment]
@rdallman Back to using the original iron_go3 api, squashed the commits

File: worker/methods.go [hunk 1]
```diff
@@ -114,6 +127,8 @@ func (w *Worker) CodePackageList(page, perPage int) (codes []CodeInfo, err error
 
 // CodePackageUpload uploads a code package
 func (w *Worker) CodePackageUpload(code Code) (id string, err error) {
```
Comment (rdallman) [inline worker/methods.go]: let's delete this func.

**Comment by rdallman** [comment]
gonna test it out in ironcli real quick make sure it all works. thanks

**Comment by rdallman** [comment]
@bg451 I think this changes too much. going to start from scratch on just porting the worker stuff over. i know some of the stuff in iron_go3 was positive

**Pull request closed without merging**